extensions/authentication-passport/tsconfig.json

```diff
@@ -7,9 +7,13 @@
     "composite": true
   },
   "include": [
-    "src"
+    "src",
+    "../../fixtures/mock-oauth2-provider/src/mock-oauth2-social-app.ts"
```

Comment:
This is wrong. We should list `@loopback/mock-oauth2-provider` as devDependency.

Reply:
agreed